src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -221,11 +233,354 @@ function svdTTN(
   return H
 end
 
+function qn_svdTTN(
+  os::OpSum{C}, sites::IndsNetwork{VT,<:Index}, root_vertex::VT; kwargs...
+)::TTN where {C,VT}
+  # Function barrier to improve type stability
+  ValType = ITensors.determineValType(terms(os))
+  return qn_svdTTN(ValType, os, sites, root_vertex; kwargs...)
+end
+
+function qn_svdTTN(
+  ValType::Type{<:Number},
+  os::OpSum{C},
+  sites::IndsNetwork{VT,<:Index},
```

Comment:
```suggestion
  sites::IndsNetwork,
```

Reply:
Also replace uses of `VT` with `sites_vertextype` and define `sites_vertextype = vertextype(sites)` at the top of the file.

Reply:
the parametric type `VT` is also used in another positional arg, `root_vertex::VT` --- should we remove it completely?

Reply:
References to `VT` have been removed, so we can simply replace `root_vertex::VT` by `root_vertex` in the signature.

Reply:
(Maybe could use something shorter like `vtype` or something like that.)